docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -40,7 +41,7 @@ $ unzip {value-name-project}.zip
 $ oc new-project {value-name-project}
 ----
 
-. Navigate to the root directory of your booster.
+. Navigate to the root directory of your project.
```

Comment:
As suggested earlier, please check if it is a "project" or an "application".

Reply:
Will do.